frontend/src/scenes/trends/personsModalLogic.tsx

```diff
@@ -78,14 +86,17 @@ export function parsePeopleParams(peopleParams: PeopleParamType, filters: Partia
 
     // If breakdown type is cohort, we use breakdown_value
     // If breakdown type is event, we just set another filter
+    const flattenedPropertyGroup = isPropertyGroup(params.properties)
```

Comment:
Thoughts on moving the isPropertyGroup check into `convertPropertyGroupToProperties`? If we know at this point we need flattened properties we can just let the function determine how to handle. Will reduce repetition

Reply:
done